CI/physmon/summary.py

```diff
@@ -5,39 +5,34 @@
 import re
 import functools
 import os
+import csv
 
 HERALD_URL = "https://herald.dokku.paulgessinger.com/view/{repo}/runs/{run_id}/artifacts/{artifact_name}/{path}"
 IS_CI = "GITHUB_ACTIONS" in os.environ
 
 
 parser = argparse.ArgumentParser()
-parser.add_argument("inputs", nargs="+")
-parser.add_argument("--base", required=True)
+parser.add_argument("results")
 parser.add_argument("--html")
 parser.add_argument("--md")
 args = parser.parse_args()
 
 re_title = re.compile(r'<p class="title">\s*(.*)\s*<\/p>', re.RegexFlag.MULTILINE)
 re_check = re.compile(r'<a.*title="(.*)">\s*(.)\s*<\/a>', re.RegexFlag.MULTILINE)
 
-summary = {}
+summary = []
 
-for h in args.inputs:
-    with open(h, mode="r", encoding="utf-8") as f:
-        try:
-            content = f.read()
-            print(h, re_title.findall(content))
-            title = re_title.findall(content)[0]
-            checks = re_check.findall(content)
-            parsed_checks = list(map(lambda c: c[1] == "✅", checks))
-            summary[h] = {
+with open(args.results) as f:
+    reader = csv.reader(f)
+    for row in reader:
+        title, slug, ec = row
```

Comment:
```suggestion
    for title, slug, ec in reader:
```
more pythonic? (not sure)